### DISCUSSION TITLE
Github copilot paying problem

### DISCUSSION THREAD

**Discussion by Berezhnov** [community/community#19630] [Copilot Conversations] [answered]
Hey! I am trying to signup on github copilot. I tried one credit card, but it had expired date, so github copilot page showed me “There are issue with your payment, we will check it in 48 hours”.

After I saw this message, I changed card to another one, but now, after 4+ days, I see another message:

" Cannot proceed with payment

We are still investigating the issue with your GitHub account, and we will update you as soon as possible. Thank you for your patience during this process. We are sorry for any inconvenience.

"

on [Sign in to GitHub · GitHub](https://github.com/github-copilot/signup) page.

My question is how to resolve this problem? Thanks!

**Comment by D7EAD** [comment, marked as answer]
Hi!

I suggest for you to reach out to Github's Support Engineers for this issue with your payment method so they can diagnose and reset it, if necessary. You can do so at [this](https://support.github.com/contact/account?tags=docs-generic) link.

**Comment by reflectioncamps** [comment]
Today i signedup for new githubcopilot account got 30day free trail account. it checked my payment details also,charged $10 and deposited back the same amount .it's been 45min but not giving access in my VSCode IDE and in the account"We may place a temporary hold on your payment method to verify its validity. This is not a charge, and it will be released automatically after verification." -- i do not know ..whats happening there?

**Reply by reflectioncamps** [reply]
in VS code,"You've reached your monthly chat messages quota. Upgrade to Copilot Pro (30-day free trial) or wait for your allowance to renew.

Upgrade to Copilot Pro
Click to retry.
Changes may take a few minutes to take effect." it's not reflecting  the status
